sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/csv/CSVOptions.scala

```diff
@@ -167,6 +168,7 @@ private[csv] class CSVOptions(
     format.setDelimiter(delimiter)
     format.setQuote(quote)
     format.setQuoteEscape(escape)
+    format.setCharToEscapeQuoteEscaping(quote)
```

Comment:
It seems in https://github.com/uniVocity/univocity-parsers/blob/master/README.md#escaping-quote-escape-characters it sets the same `escape` for both `setQuoteEscape` and `setCharToEscapeQuoteEscaping`and I sounds correct to me. However, it seems different here. Do you mind if I ask the reason?